tests/runners/airflow/demo/liminal.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: MyDataScienceApp
```

Comment:
I suggest to move this away from the "tests" and into "examples"

Also go over the yaml and change names to more meaningful ones (instead of foo/baz and my server) to make it easier to follow